build_projects/shared-build-targets-utils/DependencyVersions.cs

```diff
@@ -7,9 +7,9 @@ namespace Microsoft.DotNet.Cli.Build
 {
     public class DependencyVersions
     {
-        public static readonly string CoreCLRVersion = "1.0.2";
-        public static readonly string JitVersion = "1.0.2";
-        public static readonly string CoreCLRVersion_Latest = "1.1.0-beta-24415-01";
-        public static readonly string JitVersion_Latest = "1.1.0-beta-24415-01";
+        public static readonly string CoreCLRVersion = "1.1.0-preview1-24508-06";
+        public static readonly string JitVersion = "1.1.0-preview1-24508-06";
+        public static readonly string CoreCLRVersion_Latest = "1.1.0-preview1-24508-06";
+        public static readonly string JitVersion_Latest = "1.1.0-preview1-24508-06";
```

Comment:
I'm also curious about the `_Latest` properties and if these can be combined. [It looks like](https://github.com/dotnet/core-setup/blob/05ce78fa94159446cb814e5fac1fa193455e7452/build_projects/shared-build-targets-utils/Utils/SharedFrameworkPublisher.cs#L45-L47) the `_Latest` is used for crossgen with `win10-arm64`, otherwise non-latest.

(Made a new comment so github doesn't hide the thread as outdated. /cc @ianhays @eerhardt )